pkg/storage/internalstorage/config.go

```diff
@@ -44,6 +44,12 @@ type LogConfig struct {
 	Colorful                  bool          `yaml:"colorful"`
 	SlowThreshold             time.Duration `yaml:"slowThreshold" default:"200ms"`
 	IgnoreRecordNotFoundError bool          `yaml:"ignoreRecordNotFoundError"`
+	Filename                  string        `yaml:"filename"`
```

Comment:
Will it be better to use these parameters in the nested structure?

Reply:
Great idea